components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIConsumerImpl.java

```diff
@@ -109,9 +109,15 @@ public Subscriber getSubscriber(String subscriberId) throws AppManagementExcepti
         return subscriber;
     }
 
+    /**
+     * Get business owner for a given business owner id.
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files